build.gradle

```diff
@@ -21,7 +31,19 @@ android {
     }
 }
 
+
 dependencies {
-    compile 'com.android.support:support-v4:26.1.0'
     compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:support-v4:27.1.1'
+    compile 'com.android.support:appcompat-v7:27.1.1'
+
+    // https://mvnrepository.com/artifact/com.google.code.gson/gson
+    compile group: 'com.google.code.gson', name: 'gson', version: '2.8.2'
+
+    compile 'io.reactivex.rxjava2:rxandroid:2.0.1'
+    // Because RxAndroid releases are few and far between, it is recommended you also
+    // explicitly depend on RxJava's latest version for bug fixes and new features.
+    compile 'io.reactivex.rxjava2:rxjava:2.1.0'
+
+    compile group: 'commons-io', name: 'commons-io', version: '2.5'
 }
```

Comment:
New line :P

Reply:
Where exactly? Or do you mean that there shouldn't be any empty lines in the dependency section? :D

Reply:
The file should end with a new line.
It is some kind of posix standard, which I am not sure if we really need it (and thus forget it very often), though it cannot be bad, to have it ;-)

http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206 
> A sequence of zero or more non- <newline> characters plus a terminating <newline> character.